app/utils/markdown.js

```diff
@@ -1,13 +1,20 @@
 // Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
-import {Platform, StyleSheet} from 'react-native';
+import {Dimensions, PixelRatio, Platform, StyleSheet} from 'react-native';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
 export function getCodeFont() {
     return Platform.OS === 'ios' ? 'Menlo' : 'monospace';
 }
 
+const portraitWidth = Dimensions.get('window').width <= Dimensions.get('window').height ? Dimensions.get('window').width : Dimensions.get('window').height;
+const scale = portraitWidth / 320;
```

Comment:
Edited this to always calculate scaled font size on the portrait view of the phone. This way we don't get rotation surprises.

Reply:
Why only for portrait, I don't follow?

Reply:
Because the font's size should always be the same. We are getting the relative scale based on the device's width. If we recalculate on landscape mode the scale would not be relative 320, so we would require a separate function.
We just want the fonts to appear the same relative size on the screen consistently.

Does that make sense? hard to explain via text. We can chat if that is better to ensure it is correct

Reply:
Make sense to me